module/VuFind/src/VuFind/I18n/Translator/Loader/ExtendedIni.php

```diff
@@ -37,218 +38,209 @@
  * @category VuFind
  * @package  Translator
  * @author   Demian Katz <demian.katz@villanova.edu>
+ * @author   Sebastian Kehr <kehr@ub.uni-leipzig.de>
  * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
  * @link     https://vufind.org Main Site
  */
-class ExtendedIni implements FileLoaderInterface
+class ExtendedIni implements RemoteLoaderInterface
 {
+    const EXTEND = '@extend';
```

Comment:
I opted to name the inheritance directive `extend` instead of `include` because it may only be used once per file.

Reply:
Sounds reasonable to me!